kore/test/Test/Kore/Repl/ParserTest.hs

```diff
@@ -0,0 +1,179 @@
+module Test.Kore.Repl.ParserTest (
```

Comment:
What is this module? Looks like old Megaparsec things.

Reply:
The repl tests used the same test harness as the parser tests, but they tested something completely different from terms, so it wasn't really possible to use the new version of the parser test harness to test the repl. Hence, we copied the old parser test harness and moved it to this directory.